### PR TITLE
added article on 0.6.0 release of webcomponents.js

### PR DESCRIPTION
**Pull request by tjsavage**

### CODE REVIEW

**Comment by zenorocha** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Comment by zenorocha** [comment]
Just realized that we don't have a default style for tables.

File: src/documents/articles/polyfills-0-6-0.html.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+---
+
+Exciting news - the 0.6.0 version of the `webcomponents.js` polyfills has been [released](https://github.com/webcomponents/webcomponentsjs/tree/v0.6.0). A lot of critical fixes are in this release - highlights are described below.
+
+<!-- Excerpt -->
+
+
+This also marks about 6 months since the polyfills were moved from the Polymer organization over to WebComponents.org. Here are some stats comparing the Polymer/platform repository activity in the six months prior to the move (the precursor to webcomponents.js), to the WebComponents/webcomponentsjs repository in the six months since its creation:
+
+---
```
Comment (zenorocha) [inline src/documents/articles/polyfills-0-6-0.html.md]: Unnecessary `<hr>`

**Comment by zenorocha** [comment]
Also it would be nice to have a more attractive title but I know this is hard for release blogposts.

**Comment by zenorocha** [comment]
Good job @tjsavage!

**Comment by tjsavage** [comment]
updated - table looks way, way nicer, thanks @zenorocha!

**Comment by zenorocha** [comment]
Ok, let me know when you want this merged and published.

**Comment by addyosmani** [comment]
Comments from my review were addressed upstream on the Polymer blog version. This version looks solid to land :+1:

**Comment by tjsavage** [comment]
we're good to publish this ASAP!

**Comment by zenorocha** [comment]
Published: http://webcomponents.org/articles/polyfills-0-6-0/

**Comment by zenorocha** [comment]
Twitter: https://twitter.com/Web_Components/status/582663528751886336

**Comment by zenorocha** [comment]
G+: https://plus.google.com/+WebcomponentsOrg/posts/5sZBfXmA1Y9

**Comment by zenorocha** [comment]
I took the opportunity to update the GitHub Release info too: https://github.com/webcomponents/webcomponentsjs/releases

**Comment by tjsavage** [comment]
Awesome thanks Zeno!

**Pull request merged**